package.json

```diff
@@ -62,6 +62,7 @@
   "devDependencies": {
     "@enact/docs-utils": "^0.4.10",
     "@enact/ui-test-utils": "^1.0.9",
-    "eslint-config-enact-proxy": "^1.0.8"
+    "eslint-config-enact-proxy": "^1.0.8",
+    "tsx": "^4.19.0"
```

Comment:
Is this module necessary? I cannot find any usage.

Reply:
This was needed when I first migrated to wdio9. Since it's not needed anymore, I removed it.